### DISCUSSION TITLE
How to turn on Multiple Directional lights inside Filament?

### DISCUSSION THREAD

**Discussion by akshay-varshney** [google/filament#6112] [Q&A]
As we have closed filament to work only on one directional light due to performance issue. Can someone please help to point out how can we turn on multiple directional lights?  please let me know.

**Comment by romainguy** [comment]
Filament currently supports only a single directional light.

**Reply by akshay-varshney** [reply]
Thanks for your response @romainguy. 
If not directional, is it possible to add multiple point lights or any other lights?

I am trying to add multiple point lights but end up facing this crash (https://github.com/google/filament/discussions/6102), can you please help to resolve this, what are we doing wrong here?

**Reply by romainguy** [reply]
You can add many point and spot lights yes. @pixelflinger would know about this crash, I think it was fixed recently?

**Reply by akshay-varshney** [reply]
@pixelflinger : Can you please help with this, as this is bit urgent for us. 
We already tried this solution: https://github.com/google/filament/pull/5322/files but this also didn't worked for us. still crashing..

**Comment by RonMehta** [comment]
@pixelflinger @romainguy  : is there an ETA for enabling multiple directional lights, Furthermore can you help us understand the technical challenges in doing so, thanks in advance.

**Reply by pixelflinger** [reply]
No, this is not planned.

There is no huge difficulty other than we've hardcoded one directional light in many places. So it's really a matter of adding a lot of plumbing.
